frontend/src/__tests__/cypress/cypress/pages/components/NIMCard.ts

```
@@ -6,15 +6,118 @@ export class NIMCard extends Card {
+      '[data-testid*="nvidia-nim"]',
+      '[id*="nvidia-nim"]',
+      '[data-testid*="nim"]',
+      '[id*="nim"]',
+      '[class*="nim"]',
+      '[aria-labelledby*="nim"]'
+    ];
+    
+    cy.log(`🔍 Trying to get NIM card with ${selectors.length} different selectors...`);
+    
+    // Try each selector in order
+    for (let i = 0; i < selectors.length; i++) {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)